### PR TITLE
Added optional --schema switch

### PR DESCRIPTION
**Pull request by mcdonnelldean**
Changes for #4

### CODE REVIEW

File: bin.js [hunk 1]
```diff
@@ -26,9 +26,9 @@ var argv = minimist(process.argv.splice(2), {
 })
 
 function usage () {
-  console.log('Usage: pes-client SCHEMA [--port PORT] [--host HOST]\n' +
-              '                  [--target-host HOST] [--target-port PORT]\n' +
-              '                  [--db PATH] [--no-repl] [--from-scratch]')
+  console.log('Usage: hypem <SCHEMA> [--schema SCHEMA] [--port PORT] [--host HOST]\n' +
```
Comment (mcollina) [inline bin.js:29]: I think `<SCHEMA>` should be `[SCHEMA]`, as it's optional

Comment (mcdonnelldean) [inline bin.js:29]: @mcollina How do I say, using that notation, that I need **either** SCHEMA or [--schema], I need one or the other, I misread and thought `<SCHEMA>` meant this.

Comment (mcollina) [inline bin.js:29]: Maybe we just need a bigger help. See https://github.com/mcollina/help-me and https://github.com/mcollina/commist

Comment (mcdonnelldean) [inline bin.js:29]: Yeah, I agree,

Let me look into this.

**Comment by mcollina** [comment]
Merging anyway :dart:

**Comment by mcdonnelldean** [comment]
Cool, I hit max bandwidth this weekend. By rights a better CLI should probably be it's own issue.

**Pull request merged**